submissions/pseudoscience/pseudoscience_non-fiction.md

```diff
@@ -0,0 +1,29 @@
+
+### Nauvis reflects at least 88.3% of all light that hits it
+
+A single solar panel has a 9 m\^2 area and produces 60 kW during daytime. From that number we find that the surface receives at least 6.67 kW/m\^2 (compare to 1 kW/m\^2 for Earth). Considering Nauvis's ambient temperature is 15 degrees Celsius, we can use the [effective temperature formula](https://en.wikipedia.org/wiki/Effective_temperature#Surface_temperature_of_a_planet) to find that if Nauvis was a perfect blackbody, it has to have an albedo of 0.883 (keeping in mind that it's tidally locked). For comparison, the Earth has an albedo of 0.306, the Moon has an albedo of 0.12, and Enceladus, the most reflective body in the Solar system, has an albedo of 0.9. And all of that without even considering that solar panels have less than 100% efficiency, and that Nauvis is not an ideal blackbody!
+
+### Nuclear fuel is 3 times hotter than the surface of the Sun
+
+Now it's time to come back from space and take a look at the artificial things. Locomotives have a constant energy consumption of 600 kW, but better fuel makes them faster. This means higher-tier fuel increases the engine's efficiency. Since useful power is force times velocity, efficiency is proportional to acceleration times top speed. Now, since we can use any fuel to power the locomotive, we can safely assume it is based on a [heat engine](https://en.wikipedia.org/wiki/Heat_engine). If we assume that the engine is a perfect heat engine that works using the [Carnot cycle](https://en.wikipedia.org/wiki/Carnot_cycle), and that coal burns at a temperature of 165 degrees Celsius (based on boiler steam temperature) and ambient temperature is 15 degrees, after plugging in all the numbers we get that nuclear fuel has a temperature of over 18000 degrees.
+
+### Nukes are severely underpowered
+
+One piece of uranium-235 can produce 10 fuel cells, each of which contains 8 GJ (let's not think about neighbor bonus). This means a single item of U-235 has 80 GJ of fission energy in it, and since one nuke uses 30 pieces of U-235, with a 20% fission rate (which is what you get with boosted fission something that is required for such a small nuke), it should have an explosive yield of 115 tons of TNT. That doesn't sound like a lot, but one of those would destroy everything completely in a 100 tile radius, would kill everything in a 250 tile radius, and would do damage in a 500 tile radius (based on [Nukemap](https://nuclearsecrecy.com/nukemap/))
```

Comment:
Could you add some more perspective here about how the nukes should work in theory vs how they actually work? An image of a factory (or maybe large biter base?) with a circle showing the impact of each would probably be enough to illustrate the point in context. Also again feel free to lengthen this to may it a more enjoyable read.